cmd/cli/cmd/alpha/content/content.go

```diff
@@ -0,0 +1,21 @@
+package content
+
+import "github.com/spf13/cobra"
+
+var manifestOutputDirectory string
+
+// NewCmd returns a cobra.Command for content generation operations.
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "content",
+		Short: "Content generation",
+		Long:  "Subcommand for various content generation operations",
+	}
+
+	cmd.AddCommand(NewInterface())
+	cmd.AddCommand(NewTerraform())
+
```

Comment:
We also need to generate InterfaceGroup. It could be a separate command or we can add a flag to generate it along Interface.

Reply:
Yeah, I forgot about it